style.css

```diff
@@ -503,3 +523,316 @@ ul.points {
 .line2 {
   margin-left: 110px;
 }
+
+.vecto {
+  display: none;
+}
+
+.Imgo {
+  display: none;
+}
+
+.imageo {
+  display: none;
+}
+
+.linee {
+  display: none;
+}
+
+.contact-form-ftext {
+  display: none;
+}
+
+.contact-form-etext {
+  height: 32px;
+  width: 300px;
+  padding: 5px 10px 5px 10px;
+  border: none;
+  border-bottom: 1px solid gainsboro;
+  font-family: 'Inter', sans-serif;
+  margin-left: 24px;
+  margin-top: 20px;
+  font-size: 15px;
+}
+
+.a4 {
+  display: none;
+}
+
+@media only screen and (min-width: 768px) {
+  .main-header {
+    width: 100%;
+    height: 80px;
+  }
+
+  .header {
+    margin-top: 7px;
+    margin-left: 80px;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  .nam {
+    display: flex;
+    width: 200px;
+  }
+
+  .men {
+    margin-left: 500px;
+    margin-top: 23px;
+    display: flex;
+    gap: 32px;
+    font-family: 'Inter', sans-serif;
+    font-weight: 600;
+  }
+
+  .mail {
+    margin-top: -10px;
+  }
+
+  .section1 {
+    width: 100%;
+    display: flex;
+    height: 870px;
+  }
+
+  .containe2 {
+    display: none;
+  }
+
+  .im {
+    display: flex;
+  }
+
+  .aaa {
+    display: flex;
+    flex-direction: row-reverse;
+  }
+
+  .aaaaa {
+    display: flex;
+    flex-direction: row-reverse;
+  }
+
+  .logo {
+    display: flex;
+    flex-direction: column;
+    margin-top: 325px;
+  }
+
+  .intro {
+    width: 450px;
+    font-size: 50px;
+    margin-top: -20px;
+    margin-left: -50px;
+  }
+
+  .profession {
+    margin-top: -25px;
+    font-size: 20px;
+    margin-left: -50px;
+  }
+
+  .services {
+    margin-left: -70px;
+    margin-top: 500px;
+    width: 400px;
+    font-size: 14px;
+  }
+
+  .vector {
+    display: none;
+  }
+
+  .Img {
+    display: none;
+  }
+
+  .container {
+    margin-left: 550px;
+  }
+
+  .vecto {
+    display: flex;
+    margin-left: 320px;
+    margin-top: -100px;
+  }
+
+  .Imgo {
+    display: flex;
+    margin-top: 100px;
+  }
+
+  .Multi-Post {
+    margin-left: 720px;
+    margin-top: -870px;
+  }
+
+  .para {
+    margin-left: 720px;
+    margin-top: -650px;
+    width: 380px;
+  }
+
+  .listo {
+    margin-left: 720px;
+    margin-top: -350px;
+  }
+
+  .work {
+    width: 100%;
+    margin-left: -50px;
+  }
+
+  .containo {
+    display: grid;
+    grid-template-columns: auto auto auto;
+    column-gap: -50px;
+    margin-left: 205px;
+  }
+
+  .container2 {
+    margin-left: -50px;
+  }
```

Comment:
- [ ] According to the Figma design there isn't a See project `button` unless when hovered upon. Kindly fix this 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/206456192-838af387-4fbc-4939-b19c-faf97e878a74.png)  |  ![image](https://user-images.githubusercontent.com/74814780/206456244-14181cf4-7f1b-47da-95d9-dba176c9d428.png)

- [ ] When one of the cards are hovered on in  your project section your design should look like this ⬇️. Kindly fix this 🙏 

![image](https://user-images.githubusercontent.com/74814780/206456681-f7e436ba-ad46-4954-956f-58ecc672cc92.png)